docker/run_mdswshell.sh

```diff
@@ -1,58 +0,0 @@
-#!/bin/bash
```

Comment:
I moved the minidump-stackwalk code to a separate repo and should have removed this file.